test/curator.sh

```diff
@@ -286,7 +286,7 @@ EOF
     os::log::info Testing curator for incorrect project name length error - updating config and rolling out . . .
     update_config_and_restart $curtest "$testschedule" errors
     # curator pod status reported an error state
-    curpod=`get_completed_pod curator${ops}`
+    curpod=`get_error_pod curator`
```

Comment:
you are removing the `${ops}` here?

Reply:
No, I added it there by mistake. Just one run for this test is sufficient, as it's a parsing error - won't touch any instance of es.

Also, this is covered by unit tests:
https://github.com/josefkarasek/origin-aggregated-logging/commit/d3f4330c2c4ebcf7dfb65754e518d8a6aed82398#diff-c70783a5f3dbb21cb91b2eca2cbebe74R38
https://github.com/josefkarasek/origin-aggregated-logging/blob/d3f4330c2c4ebcf7dfb65754e518d8a6aed82398/curator/test/name-invalid-cfg.yaml
https://github.com/josefkarasek/origin-aggregated-logging/blob/d3f4330c2c4ebcf7dfb65754e518d8a6aed82398/curator/test/name-too-long-cfg.yaml